### PR TITLE
Add a test to check element resolving

### PR DESCRIPTION
**Pull request by Hanna-Shalamitskaya-EPAM**
Changes:

- Added a test for the element resolving logic

### CODE REVIEW

**Review by Yauhenikapl — APPROVED** [review]

File: core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.models/0.2.0/AspectWithProperties.ttl [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2023 Robert Bosch Manufacturing Solutions GmbH, Germany. All rights reserved.
+#
+
+@prefix : <urn:samm:com.bosch.test.models:0.2.0#>.
+@prefix type: <urn:samm:com.bosch.test.type:0.2.0#> .
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+
+:ProcessParameters a samm:Aspect ;
```
Comment (atextor) [inline core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.models/0.2.0/AspectWithProperties.ttl:9]: Please use a neutral Aspect Model name

File: core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.models/0.2.0/AspectWithProperties.ttl [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2023 Robert Bosch Manufacturing Solutions GmbH, Germany. All rights reserved.
+#
+
+@prefix : <urn:samm:com.bosch.test.models:0.2.0#>.
```
Comment (atextor) [inline core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.models/0.2.0/AspectWithProperties.ttl:5]: The test namespace should be `urn:samm:org.eclipse.esmf.test:1.0.0#`

File: core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.models/0.2.0/AspectWithProperties.ttl [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+@prefix type: <urn:samm:com.bosch.test.type:0.2.0#> .
```
Comment (atextor) [inline core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.models/0.2.0/AspectWithProperties.ttl:6]: If this needs to be in another namspace than the default test namespace, it should be included in it, e.g. `urn:samm:org.eclipse.esmf.test.other:1.0.0#`

File: core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.models/0.2.0/Part_shared.ttl [hunk 4]
```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2023 Robert Bosch Manufacturing Solutions GmbH, Germany. All rights reserved.
+#
+
+@prefix : <urn:samm:com.bosch.test.models:0.2.0#> .
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+:processedPartId a samm:Property ;
```
Comment (atextor) [inline core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.models/0.2.0/Part_shared.ttl:10]: Please use a neutral model element name

File: core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.models/0.2.0/Part_shared.ttl [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    samm:preferredName "Processed Part ID"@en ;
```
Comment (atextor) [inline core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.models/0.2.0/Part_shared.ttl:11]: Adapt to neutral element name

File: core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.models/0.2.0/Part_shared.ttl [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    samm:description "The identification of the processed part based on part number."@en ;
```
Comment (atextor) [inline core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.models/0.2.0/Part_shared.ttl:12]: Remove

File: core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.models/0.2.0/Part_shared.ttl [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+    samm:exampleValue "F03Z1234560213107000283011234567990" ;
```
Comment (atextor) [inline core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.models/0.2.0/Part_shared.ttl:13]: Remove

File: core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.models/0.2.0/Part_shared.ttl [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+    samm:characteristic :PartNumber .
+
+:PartNumber a samm-c:Trait ;
+    samm:see <https://inside-docupedia.bosch.com/confluence/x/ssq0O> ;
```
Comment (atextor) [inline core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.models/0.2.0/Part_shared.ttl:17]: Remove

File: core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.type/0.2.0/Material_shared.ttl [hunk 9]
```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2023 Robert Bosch Manufacturing Solutions GmbH, Germany. All rights reserved.
+#
+
+@prefix : <urn:samm:com.bosch.test.type:0.2.0#>.
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+:materialNumber a samm:Property ;
```
Comment (atextor) [inline core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.type/0.2.0/Material_shared.ttl:10]: Please use neutral model element name

File: core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.type/0.2.0/Material_shared.ttl [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+    samm:preferredName "Material Number"@en ;
```
Comment (atextor) [inline core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.type/0.2.0/Material_shared.ttl:11]: Remove

File: core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.type/0.2.0/Material_shared.ttl [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+    samm:preferredName "Materialnummer"@de ;
```
Comment (atextor) [inline core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.type/0.2.0/Material_shared.ttl:12]: Remove

File: core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.type/0.2.0/Material_shared.ttl [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+    samm:description "The identification of the product based on the SAP material number."@en ;
```
Comment (atextor) [inline core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.type/0.2.0/Material_shared.ttl:13]: Remove

File: core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.type/0.2.0/Material_shared.ttl [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+    samm:exampleValue "F03Z12345602" ;
```
Comment (atextor) [inline core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.type/0.2.0/Material_shared.ttl:14]: Remove

**Review by atextor — CHANGES_REQUESTED** [review]

**Review by atextor — APPROVED** [review]

**Pull request merged**